src/main/java/com/commercebank/controller/AppointmentController.java

```diff
@@ -60,6 +62,11 @@ public AppointmentController(AppointmentDAO appointmentDAO, CustomerDAO customer
         return appointmentDAO.list();
     }
 
+    @RequestMapping(value = "/delete/{id}/", method = RequestMethod.GET)
```

Comment:
Is there a reason you changed this from a DELETE request?

Reply:
No, delete didn't work either so I was checking to see if GET worked. I just changed it back to DELETE.